src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkVideoPlayer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from "react"
+import { Box, ResponsiveBox } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtworkVideoPlayer_artwork } from "v2/__generated__/ArtworkVideoPlayer_artwork.graphql"
+
+interface ArtworkVideoPlayerProps {
+  artwork: ArtworkVideoPlayer_artwork
+  small?: boolean
+}
+
+const ArtworkVideoPlayer: FC<ArtworkVideoPlayerProps> = ({
+  artwork: { video },
+  small,
+}) => {
+  if (!video) {
+    return null
+  }
+
+  return (
+    <Box
+      my={2}
+      width="100%"
+      // This minHeight works for the 10 POC video items
+      // Should be replaced with a dynamic value in the future
+      minHeight={small ? "inherit" : 800}
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+    >
+      <ResponsiveBox
+        bg="black10"
+        mx={[0, 2]}
+        // @ts-ignore
+        maxWidth={video.width ?? "100%"}
```

Comment:
Do we want these videos to fill the available box or be constrained by their Vimeo dimensions? I'd imagine fill — in which case this should be `100%`

Reply:
I thought that too, but they get _really_ grainy. Full screen option is available and gives a better image. IDK... I'll post up a side by side and see what you think.